### PR TITLE
add myusdc app

### PR DESCRIPTION
**Pull request by jazz-cb**

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: app/page.tsx [hunk 1]
```diff
@@ -44,6 +44,12 @@ export default function Home() {
 
     <div className="grid grid-cols-2 md:grid-cols-2 lg:grid-cols-2 gap-10 max-w-6xl mx-auto">
     {[
+        { 
+          name: 'USDC Payments App', 
+          route: 'https://www.myusdc.app/',
+          description: 'Send testnet USDC to any address in a familiar banking interface',
+          buildLink: 'https://discord.gg/cdp'
```
Comment (murrlincoln) [inline app/page.tsx:51]: https://github.com/HeimLabs/coinbase-myusdc-backend

**Review by murrlincoln — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for murrlincoln @ 2024-10-02 16:42:48 UTC
User must have write permissions to review

File: app/page.tsx (hunk 1 above)
Comment (jazz-cb) [reply app/page.tsx]: fixed

**Review by rohan-agarwal-coinbase — APPROVED** [review]

**Pull request merged**